base/random.jl

```diff
@@ -1194,6 +1196,9 @@ The `Base` module currently provides an implementation for the types
     end
 end
 
+Base.@irrational SQRT_HALF 0.7071067811865475244008  sqrt(big(0.5))
+randn{T}(rng::AbstractRNG, ::Type{Complex{T}}) = Complex{T}(SQRT_HALF * randn(rng, T), SQRT_HALF*randn(rng, T))
```

Comment:
For new code, maybe use the `where` syntax. Also, don't you want to restrict `T<:AbstractFloat`?